src/seedu/addressbook/commands/viewAttendanceCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Attendance;
+
+/**
+ *  Lists down the dates where the person's attendance has been taken.
+ */
+public class viewAttendanceCommand extends Command {
+    public static final String COMMAND_WORD = "viewAtten";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Views the attendance of a student. \n"
+            + "Parameters: n/STUDENTINDEXn\t"
+            + "Example: " + COMMAND_WORD + " "
+            + "n/Neow Bo Kai";
+
+    public static final String MESSAGE_SUCCESS = "Attendance for student: ";
+    public static final String MESSAGE_DUPLICATE_EXAM = "No such student";
+
+    // Constructor
+    public viewAttendanceCommand(String indexOfPerson){}
+
+    @Override
+    public CommandResult execute() {
+
+            return new CommandResult("Attendance of student: " + Attendance.viewAttendance());
```

Comment:
see above for example on how to change. you cannot call methods in Attendance.class because you are not even linking it to any person.